spec/functional/compile_message_spec.rb

```diff
@@ -0,0 +1,34 @@
+RSpec.describe "compiling a simple message" do
+  before :context do
+    require compile("sally.proto")
```

Comment:
compiling and removing the constants is a bit heavy handed in this tests, but it is the fastest way I have found to get feedback on this process. Currently running the full test suite takes ~1.8sec wall clock time